heron/api/src/java/com/twitter/heron/streamlet/impl/utils/StreamletUtils.java

```diff
@@ -0,0 +1,27 @@
+//  Copyright 2017 Twitter. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+package com.twitter.heron.streamlet.impl.utils;
+
+public final class StreamletUtils {
```

Comment:
I think it might be a bit premature to have a separate utils class for the streamlet API. I like this `require` function but I'd say let's keep it in the `StreamletImpl` class for now. If a handful of utility functions emerge then a util class will become necessary.

Reply:
Sure, this has just been addressed with last commit.